### ISSUE TITLE
Controller Input

### ISSUE THREAD

**Issue by zhefan** [ctallec/world-models#36]
Hi, I wonder if the input to the controller should be the _latent vector_ (z) from VAE and hidden vector from RNN?

https://github.com/ctallec/world-models/blob/d6abd9ce97409734a766eb67ccf0d1967ba9bf0c/utils/misc.py#L161

But the code here shows that one of the inputs is the gaussian mean instead.

**Comment by pdeubel** [comment]
Yes I though the same, as it turns out there was already a discussion about this in #28. Seems like it does not affect the performance of that particular environment, nevertheless I think the correct latent vector `z` should be used

**Comment by brettmartin4** [comment]
I can't say for sure about the car racing experiment, but for other environments, using the mean will produce erroneous behavior.
